### ISSUE TITLE
Url not working

### ISSUE THREAD

**Issue by keitje** [serviceprototypinglab/dcvalidator#12]
Hi, your URL http://160.85.252.231:3000/ is not working anymore.

**Comment by alidaghighi** [comment]
Yes unfortunately it was a demo and the server is down but I'll update the new URL, you can follow up [here](https://github.com/alidaghighi/dcvalidator)
